### PR TITLE
TVAULT-3572: avoiding our own nova sudoers entry

### PR DESCRIPTION
**Pull request by abhijeetpatra**
- We ll use the default nova sudoers permission

### CODE REVIEW

**Comment by abhijeetpatra** [comment]
Cleaned up dmapi charm as well.
 - no user or group creation
 - no ownership change of /etc/dmapi/* or /var/cache/dmapi or /var/log/dmapi

File: juju-charms/charm-trilio-data-mover/src/files/trilio/trilio_sudoers [hunk 1]
```diff
@@ -1 +0,0 @@
-nova ALL = (root) NOPASSWD: /home/tvault/.virtenv/bin/privsep-helper *
```
Comment (amitkumar-a4) [inline juju-charms/charm-trilio-data-mover/src/files/trilio/trilio_sudoers]: @abhijeetpatra Are we removing this?

Comment (abhijeetpatra) [reply juju-charms/charm-trilio-data-mover/src/files/trilio/trilio_sudoers]: yes, when s3 package would be provided, the sudoers entry would be a part of that package.

Comment (amitkumar-a4) [reply juju-charms/charm-trilio-data-mover/src/files/trilio/trilio_sudoers]: datamover package will also have this entry or only S3 package will have this?

We need this entry in datamover package.

Comment (abhijeetpatra) [reply juju-charms/charm-trilio-data-mover/src/files/trilio/trilio_sudoers]: do we need the privsep for nfs as well ?

Comment (amitkumar-a4) [reply juju-charms/charm-trilio-data-mover/src/files/trilio/trilio_sudoers]: yes path related activities we are performing via privsep. https://github.com/trilioData/contego/blob/master/contego/privsep/path.py

Comment (abhijeetpatra) [reply juju-charms/charm-trilio-data-mover/src/files/trilio/trilio_sudoers]: Resolving this one, as we are using system's privsep helper and its suders permission are already set by nova. I ll check in the contego installation script as well.

**Pull request merged**